PluginCore/PluginCore/Controls/CompletionList.cs

```diff
@@ -19,37 +19,25 @@ public class CompletionList
         /// <summary>
         /// Properties of the class 
         /// </summary> 
-        private static System.Timers.Timer tempo;
-        private static System.Timers.Timer tempoTip;
-        private static System.Windows.Forms.ListBox completionList;
+        internal static CompletionListControl completionList;
```

Comment:
We probably want to drop this wrapper class if it only forwards calls to the instance.